src/api/files/files.js

```diff
@@ -41,11 +41,7 @@ export default class Files {
 
             const response = await http(
                 fetch,
-                {
-                    timeout: this.config.timeout,
-                    agent: this.config.agent,
-                    formData: true,
-                },
+                { ...this.config.timeout, formData: true },

```

Comment:
Did you mean to destructure *this.config*?

Reply:
Thanks! Updated